src/Site_Command.php

```diff
@@ -651,6 +651,82 @@ public function unspam( $args ) {
 		$this->update_site_status( $args, 'spam', 0 );
 	}
 
+	/**
+	 * Set one or more sites to mature.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <id>...
+	 * : One or more IDs of sites to set as mature.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     $ wp site mature 123
+	 *     Success: Site 123 set to mature.
+	 *
+	 * @subcommand mature
+	 */
+	public function mature( $args ) {
+		$this->update_site_status( $args, 'mature', 1 );
+	}
+
+	/**
+	 * Set one or more sites for general audience.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <id>...
+	 * : One or more IDs of sites to set for general audience.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     $ wp site general 123
+	 *     Success: Site 123 marked for general audience.
+	 *
+	 * @subcommand general
+	 */
+	public function general( $args ) {
```

Comment:
For parity with the other commands, let's call this `unmature` instead of `general`. As it stands, it's not clear that `general` is the opposite of `mature`.